### ISSUE TITLE
2D PhysicsMaterial bounce > 0, constantly agitated/vibrating when they should be at rest

### ISSUE THREAD

**Issue by hedgehog90** [godotengine/godot#25186]
**Godot version:**
3.1.beta2.mono.official

**Issue description:**
If you place lots of objects in a physics environment  with >0 bounce, they vibrate and never fully come to rest.
I think setting the gravity higher makes it even worse.

You can see this demonstrated in the 'Scene Instancing Demo' from the official demos repo (misc/instancing).
It doesn't make sense that a ball with <1 bounce never comes to rest, there must be some strange and unnecessary impulse being applied behind the scenes.

**Comment by avencherus** [comment]
I was having the same problem:  https://github.com/godotengine/godot/issues/24332

**Comment by gianllucah** [comment]
Same problem here.

**Comment by hirsimaki-markus** [comment]
I have found a somewhat reliable workaround for this. The ball in below game can only jump if its colliding e.g. not agitated.

![quantum-yeet](https://user-images.githubusercontent.com/46528769/78435132-ea795b00-767e-11ea-957f-d6caabf893eb.gif)

I confirmed it to work with high weights like 100. Values like weight and bounciness might require tweaking along with the threshold below but this should stop infinite bouncing:

```gdscript
func _physics_process(delta):
	if abs(linear_velocity.y) <= 100:
		physics_material_override.absorbent = 1
	else:
		physics_material_override.absorbent = 0
```

I have no idea how to contribute to the source but I suppose the above logic could be implemented directly into source.
